Scripts/Uninstall.ps1

```diff
@@ -0,0 +1,40 @@
+<#
+
+.SYNOPSIS
+Uninstalls the Hyper game.
+
+.DESCRIPTION
+This script is meant to be invoked from the installation folder of the game,
+i.e. the unzipped release folder.
+
+.EXAMPLE
+
+PS> ./Uninstall.ps1
+
+#>
+
+$scriptDirectory = Split-Path -Parent $MyInvocation.MyCommand.Definition
```

Comment:
This script is super dangerous. I don't think it should delete all objects in a folder. Just some objects with predefined names. i.e. Hyper.exe, uninstall.ps1 stuff like that. Also this should be an exe not a powershell script

Reply:
I agree that it can be dangerous, but it's hard to be 100% safe here.
We could do something like this: we could add a .txt to the zip containing names and creation dates of the files.
The uninstall script would then compare this information with that it's currently removing to make sure we're not removing something else

Reply:
Ok, better safe than sorry, I removed this script